common/src/test/java/com/microsoft/identity/common/internal/AccountManagerBrokerDiscoveryUtilTests.kt

```diff
@@ -58,6 +60,34 @@ class AccountManagerBrokerDiscoveryUtilTests {
         Assert.assertNull(util.getActiveBrokerFromAccountManager())
     }
 
+    @Test
+    fun testGetCurrentActiveBrokerWithDebugAuthApp() {
+        val util = AccountManagerBrokerDiscoveryUtil(
+            knownBrokerApps = BrokerData.allBrokers,
+            isSignedByKnownKeys = { brokerData ->
+                // Mock validation.
+                brokerData == BrokerData.debugMicrosoftAuthenticator
+            },
+            getAccountManagerApps = {
+                getMockedAccountManager()?.authenticatorTypes!!
+            })
+        Assert.assertNotNull(util.getActiveBrokerFromAccountManager())
+    }
+
+    @Test
+    fun testGetCurrentActiveBrokerWithReleaseAuthApp() {
+        val util = AccountManagerBrokerDiscoveryUtil(
+            knownBrokerApps = BrokerData.prodBrokers,
+            isSignedByKnownKeys = { brokerData ->
+                // Mock validation.
+                brokerData == BrokerData.prodMicrosoftAuthenticator
+            },
+            getAccountManagerApps = {
+                getMockedAccountManager()?.authenticatorTypes!!
```

Comment:
nit: can we return the list of AuthenticatorDescription directly? (as opposed to setting up mockito.

Reply:
I tried doing that, if this is what you meant 
   getAccountManagerApps = {
                accountManager.authenticatorTypes
            })

But, this was not setting the authenticator.pkgName field. And in the method to get the valid broker, we are checking if authenticator.pkgName is null. We simply return. So I had to mock it.